### PR TITLE
Added support for contao core-bundle late bootstrapping of legacy code

### PR DESCRIPTION
**Pull request by Toflar**
Belongs to contao/core-bundle#44.

### CODE REVIEW

**Comment by discordier** [comment]
@Toflar I added 317e8de via https://github.com/Toflar/contao/pull/1, could you please merge so this PR will get updated?

**Comment by Toflar** [comment]
Merged.

**Comment by discordier** [comment]
Thanks. :)

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-    mailer_password:   ~
+    mailer_transport:      mail
+    mailer_host:           127.0.0.1
+    mailer_user:           ~
+    mailer_password:       ~
 
-    secret:            ThisTokenIsNotSoSecretChangeIt
+    secret:                ThisTokenIsNotSoSecretChangeIt
+
+    add_language_to_url:   true
```
Comment (aschempp) [inline app/config/parameters.yml.dist]: Do we enable this by default now? It was not in Contao 3.x

Comment (leofeyer) [inline app/config/parameters.yml.dist]: You are right. I'll change it.

**Pull request merged**